BREAKING_CHANGE_AND_DEPRECATION.md

```diff
@@ -0,0 +1,45 @@
+# Breaking changes and deprecations
+
+Breaking changes are defined as a change to any of the following that causes installation errors or 
+unexpected runtime behavior after upgrading to the next stable minor version of Ratify:
+- Verification API
+- Verification result and schema
+- Default configuration value
+- User facing plugin interfaces
+
+## Stability levels
+- Generally available features should not be removed from that version or have its behavior significantly changed to avoid breaking existing users.
+- Beta or pre-release versions may introduce breaking changes 2 release after deprecation notice.
```

Comment:
discussed in community meeting, since we are a young project, lets start with breaking changed allowed in beta.